src/ui/ToggleButton.tsx

```diff
@@ -5,7 +5,7 @@ import { Icon, IconNames } from '@nypl/design-system-react-components';
 import Button from './Button';
 
 export interface ToggleButtonProps
-  extends React.ComponentPropsWithRef<typeof ChakraBox> {
+  extends React.ComponentPropsWithoutRef<typeof ChakraBox> {
```

Comment:
was this a thing to make it build, or do toggle buttons explicitly disallow refs?

Reply:
It doesn't currently accept a ref because we don't use `React.forwardRef`, but if we change it to use that then we can accept a ref here